masterlist.yaml

```diff
@@ -151,7 +150,9 @@ plugins:
     msg:
       - type: warn
         content: 'You need to check your Tale of Two Wastelands load order'
-        condition: 'file("TaleOfTwoWastelands.esm") and not checksum("TaleOfTwoWastelands.esm", 26497E29)'
+        condition: 'file("TaleOfTwoWastelands.esm") and not checksum("TaleOfTwoWastelands.esm", 6D6F2EA7)'
```

Comment:
Sure, but check for either checksum then. :) Something like `… and not (checksum(…) or checksum(…))` might work. Perhaps request the mod author to start add a version string in the mods description metadata field so we can check against that rather than checksums in the future?